.circleci/config.yml

```diff
@@ -66,19 +66,17 @@ workflows:
       - build:
           filters:
             branches:
-              ignore: master
+              ignore:
+                - master
```

Comment:
Do we want to keep ignoring `master` here, or just ignore `release`?

Reply:
Ack, somehow I missed your comment here. I guess I figured we didn’t need to test master if we already do every other branch before merging. But I guess it’s not a bad idea in case someone lands something directly on master for some reason. I’ll change it.